languages/clojure/exercises/concept/basic-functions/.doc/instructions.md

```diff
@@ -0,0 +1,15 @@
+In this exercise you'll be creating some functions which work with the number 42, sometimes referred to as "the answer to the ultimate question of life, the universe."
+
+You have three tasks.
```

Comment:
Quoting the [documentation](https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md#step-2-add-docsinstructionsmd):

> It should explicitly explain what the student needs to do (define a method with the signature X(...) that takes an A and returns a Z), and provide at least one example usage of that function. If there are multiple tasks within the exercise, it should provide an example of each.

Could you perhaps add examples for each of the tasks?

Reply:
@ErikSchierboom I updated this to be more specific about the exact name of the method and arguments. Is this more what you had in mind?

Reply:
@JimLynchCodes Ehm, not exactly :) There is another Clojure exercise that has the examples we are looking for: https://github.com/exercism/v3/blob/master/languages/clojure/exercises/concept/lists/.docs/instructions.md